### DISCUSSION TITLE
[1174180]荒野大镖客2（仅本体）

### DISCUSSION THREAD

**Discussion by groundwater1203** [pjy612/SteamManifestCache#74] [Share]
appid:1174180
sai code:E7941DD55F7DC929
[Red Dead Redemption 2.zip.zip](https://github.com/pjy612/SteamManifestCache/files/13695394/Red.Dead.Redemption.2.zip.zip)
不太会用github,不知道格式对不对
邮箱：3588956488@qq.com
作者的软件很适合我们这种不想玩学习版大作，但没钱买正版的steam白嫖党
求一个内测名额 ！！！ QwQ

**Comment by pjy612** [comment]
查收邮件

**Reply by pjy612** [reply]
实际这还是某种学习版。。。2333
